contrib/babelfishpg_common/sql/upgrades/babelfishpg_common--3.1.0--3.2.0.sql

```diff
@@ -6,5 +6,17 @@ SELECT set_config('search_path', 'sys, '||current_setting('search_path'), false)
 DROP CAST IF EXISTS(NUMERIC AS sys.BIT);
 CREATE CAST (NUMERIC AS sys.BIT) WITH FUNCTION sys.numeric_bit (NUMERIC) AS IMPLICIT;
 
+DROP CAST IF EXISTS(sys.VARCHAR AS sys.MONEY);
+
+DROP FUNCTION IF EXISTS sys.varchar2money(sys.VARCHAR) CASCADE;
```

Comment:
(Blocking): Let's create the relevant babelfish_drop_deprecated_* function in this upgrade script and use it to deprecate and drop this function since a simple drop might create upgrade issues due to dependencies.